### DISCUSSION TITLE
[3.6.0] (2021-06-22)

### DISCUSSION THREAD

**Discussion by vishalnarkhede** [GetStream/stream-chat-react-native#720] [Q&A]
This release provides full support for Expo 41 and React Native 0.64

Please make sure to upgrade following packages to specified versions:

- `react-native-reanimated@2.1.0`
- `react-native-share@6.0.1`

<hr /><em>This discussion was created from the release <a href='https://github.com/GetStream/stream-chat-react-native/releases/tag/v3.6.0'>[3.6.0] (2021-06-22)</a>.</em>

**Comment by taschik** [comment]
When installing via npm I am receiving the following npm error:

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.6" from stream-chat-react-native-core@3.6.1
npm ERR! node_modules/stream-chat-react-native-core
npm ERR!   stream-chat-react-native-core@"3.6.1" from the root project
```

Shouldn't full RN0.64 support also accept react@17.x for dependencies?

The only way to solve the issue is to install with `--legacy-peer-deps`

**Reply by vishalnarkhede** [reply]
Hey @taschik thanks for brining this to our attention!

**Comment by evanserickson** [comment]
If you guys provide support for expo, can we include an expo install guide?
